src/pages/home/NewHomePage/BuiltForSection/BuiltForBlobAndImage.tsx

```diff
@@ -0,0 +1,71 @@
+import { ThemeContext } from 'contexts/Theme/ThemeContext'
+import useMobile from 'hooks/Mobile'
+import Image from 'next/image'
+import { useContext, useEffect, useState } from 'react'
+
+const Y_OFFSET_CONSTANT = 650 // If anything change above the blob/image, this needs to be updated
+
+export function BuiltForBlobAndImage({
+  card,
+}: {
+  card: 'daos' | 'crowdfunding' | 'nfts' | 'builders'
+}) {
+  const { forThemeOption } = useContext(ThemeContext)
+  const isMobile = useMobile()
+
+  const [scrollPosition, setScrollPosition] = useState(0)
+
+  // Handle parallax effect on scroll
+  useEffect(() => {
+    const handleScroll = () => {
+      setScrollPosition(window.pageYOffset + Y_OFFSET_CONSTANT)
+    }
+
+    window.addEventListener('scroll', handleScroll)
```

Comment:
need to debounce this a bit probably

Reply:
Won't that make it laggy?

Reply:
Good catch. Added one and everything looks the same to me. WDYT?